README.md

```diff
@@ -4,5 +4,41 @@ This repository contains the source code for Zulip's PyPI packages:
 
 * `zulip`: [PyPI package](https://pypi.python.org/pypi/zulip/)
   for Zulip's API bindings.
-* `zulip_bots`: PyPI package for Zulip's bots and bots API.
-* `zulip_botserver`: PyPI package for Zulip's Flask bot server.
+* `zulip_bots`: [PyPI package](https://pypi.python.org/pypi/zulip-bots)
+  for Zulip's bots and bots API.
+* `zulip_botserver`: [PyPI package](https://pypi.python.org/pypi/zulip-botserver)
+  for Zulip's Flask bot server.
+
+## Development
+
+1. Fork and clone the Git repo:
+   `git clone https://github.com/<your_username>/python-zulip-api.git`
+
+2. Make sure you have [pip](https://pip.pypa.io/en/stable/installing/)
+   and [virtualenv](https://virtualenv.pypa.io/en/stable/installation/)
+   installed.
+
+3. `cd` into the repository cloned earlier:
+   `cd python-zulip-api`
+
+4. Run:
+   ```
+   ./tools/provision
```

Comment:
I think we probably want to teach users how this works, but maybe we should have `provision` print out `source` command when it finishes so you only need to remember that part?